game/static/game/css/board.css

```
@@ -1142,3 +1142,37 @@ body {
+    z-index: 1000;
+    animation: floatUp 2s ease-out forwards;
+}
+.floating-emote.white-emote {
+    left: 10%;
+    filter: drop-shadow(0 0 8px rgba(255, 255, 255, 0.8));
+}
+.floating-emote.black-emote {
+    right: 10%;
+    filter: drop-shadow(0 0 8px rgba(0, 100, 255, 0.8));
+}
+@keyframes floatUp {
```

Stylelint (17.11.0)

[error] Expected keyframe name "floatUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)